index.html

```diff
@@ -437,13 +444,91 @@ <h2>The <dfn>WOT</dfn> namespace</dfn></h2>
+        <li>Search for missing required properties in |td| accordingly to
+          <a href="https://www.w3.org/TR/2020/WD-wot-thing-description11-20201124/#json-schema-for-validation">TD JSON
+            Schema</a>.
+          (TODO: possibly expand this step)
+        </li>
+        <li>For each |missing| property run these sub-steps:
+          <ol>
+            <li>If |missing| is <code>title</code> generate a runtime unique name and assign to <code>title</code>.</li> 
+            <li>If |missing| is <code>@context</code> assign the latest supported Thing Description context URI.</li>
+            <li>If |missing| is <code>instance</code> assign the string <code>1.0.0</code>.</li>
+            <li>If |missing| is <code>forms</code> generate a list of <a>Forms</a> using the available <a>Protocol Bindings</a> and content types
+              encoders. Then assign the obtained list to <code>forms</code>. (TODO: expand?)</li>
+            <li>If |missing| is <code>security</code> assign the label of the first supported <a>SecurityScheme</a> in <code>securityDefinitions</code> field. 
+            If no <a>SecurityScheme</a> is found generate a <a>NoSecurityScheme</a> called <code>nosec</code> and assing the string <code>nosec</code>
+          to <code>security</code>. (TODO: ask for guidance to Security task force)  </li>
+            <li>If |missing| is <code>href</code> define |formStub| as the partial <a>Form</a> that does not have <code>href</code>. Generate a valid |url:URL| using the first <a>Protocol Binding</a> 
+              that satisfy the requirements of |formStub|. Assign |url| to <code>href</code>. If not <a>Protocol Binding</a> can be found remove |formStub| from |td|. </li>
+            <li>Otherwise [= exception/throw =] {{SyntaxError}} and abort these steps.
+              (TODO: we could check for this condition in the validation)</li>
+            <li>Add |missing| to |td| with |value| as value</li>
+          </ol>
+        </li>
+        <li>Run <a>validate a TD</a> on |td|. If that fails re-[= exception/throw =] the error and abort these steps</li>
+        <li>Return |td|</li>
+      </ol>
+    </section>
+    <section>
+      <h3>Validating an ExposedThingInit</h3>
+      To <dfn>validate an ExposedThingInit</dfn> given |init:ExposedThingInit|, run the following steps:
+      <ol>
+        <li>
+          Parse <a href="https://www.w3.org/TR/2020/WD-wot-thing-description11-20201124/#json-schema-for-validation">TD JSON
+            Schema</a>
+          and load it in object called |schema|
+        </li>
+        <li>
+          For each property |key| in |schema|,
+          <ol>
+            <li>If |key| equals <code>required</code> then remove |key| from |schema|</li>
+          </ol>
+        </li>
+        <li>Return the result of applying |schema| to |init|</li>
+      </ol>
+    
+      TODO: is there any property that we should still require? for example flow property is mandatory for oAuth Security
```

Comment:
Instead of the TODO, please use a `<p class="issue"> </p>` or `<p class="ednote"> </p>`